pycaret/internal/plots/time_series.py

```diff
@@ -450,14 +450,36 @@ def plot_acf(
     data_kwargs: Optional[Dict] = None,
     fig_kwargs: Optional[Dict] = None,
 ) -> PlotReturnType:
-    """Plots the ACF on the data provided"""
-    fig, return_data_dict = None, None
-
+    """Plots the ACF on the data provided
+
+    Parameters
+    ----------
+    data : pd.Series
+        Data whose correlation plot needs to be plotted
+    fig_defaults : Dict[str, Any]
+        The defaults dictionary containing keys for "width" and "height" (mandatory)
+    model_name : Optional[str]
+        The name of the model, by default None
```

Comment:
I would clarify here, If ACF is for model residuals, model_name must be passed. If model_name is `None`, then name is derived from `data`.